packages/apps/types/src/serializer/clone.ts

```diff
@@ -5,17 +5,19 @@
 import { getSchema, type ExpandoType, getEchoObjectAnnotation } from '@dxos/echo-schema';
 import { invariant } from '@dxos/invariant';
 
-import { TypeOfExpando } from './file-serializer';
+import { TypeOfExpando } from './object-serializer';
+import { jsonSerializer } from './serializer';
 import { serializers } from './serializers';
 
 /**
  * @deprecated Workaround for ECHO not supporting clone.
  */
-export const clone = async (object: ExpandoType) => {
+// TODO(burdon): Remove?
```

Comment:
yes, this should be removed as soon as echo has cloning supported natively per the jsdoc